BootstrapScriptWithToken/bootstrap.py

```diff
@@ -11,11 +11,17 @@
 from SysdbHelperUtils import SysdbPathHelper
 import Cell
 import urlparse
+from EapiClientLib import EapiClient
 
 
 ############## USER INPUT #############
 cvAddr = ""
 enrollmentToken = ""
+# currentTimeDate format hh:mm:ss mm/dd/yyy or hh:mm:ss yyyy-mm-dd. Enclosed in double quotes
+currentTimeDate = ""
+# timezone PST8PDT MST7MDT CST6CDT EST5EDT are valid US Timezones. Default PST8PDT
+# Rest of the world check switch CLI. Config>clock timezone?
+set_timezone = "PST8PDT" 
```

Comment:
we should take this as a parameter too, hence keep it empty just like other params